css/style.css

```diff
@@ -0,0 +1,84 @@
+/**
+ * Place your custom styles here.
+ */
+
+ /*GLOBAL OVERRIDES*/
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+@import url('https://fonts.googleapis.com/css?family=Oswald');
```

Comment:
OK, but at this point in time, you're not using Oswald.  The problem with merging this as it is, is that there is a chance that nobody will ever remove Oswald, and then you're loading a font (in production) for no reason.  Every PR should represent a sensible and complete view of the project at that time, with no loose ends, so I'd really prefer that Oswald is removed (and not just commented out).  If you start using it again, then add it back in.